public/js/action-list.js

```diff
@@ -82,14 +82,26 @@
                 $list.find('.list-item.active').removeClass('active');
                 $item.addClass('active');
             }
+
+            // For items that do not have a bottom status bar like Downtimes, Comments...
+            if (! $list.closest('.container').find('.footer').length) {
```

Comment:
You should probably also use `children` instead of `find`. The latter traverses the entire DOM tree down, which may lead to false-positives.